discovery_engine_core/core/src/storage/sqlite.rs

```diff
@@ -91,7 +96,40 @@ impl SqliteStorage {
         Ok(Self { pool })
     }
 
-    async fn begin_tx(&self) -> Result<Transaction<'_, Sqlite>, Error> {
+    async fn setup_keep_stacks_in_sync(tx: &mut Transaction<'_, Sqlite>) -> Result<(), Error> {
+        let expected_ids = &[
+            stack::ops::breaking::BreakingNews::id(),
+            stack::ops::personalized::PersonalizedNews::id(),
+            stack::ops::trusted::TrustedNews::id(),
+            stack::exploration::Stack::id(),
+        ];
+
+        let mut query_builder = QueryBuilder::new(String::new());
+        query_builder
+            .push("INSERT INTO Stack (stackId) ")
+            .push_values(expected_ids, |mut stm, id| {
+                stm.push_bind(id);
+            })
+            .push("ON CONFLICT DO NOTHING")
```

Comment:
:art: space after the closing bracket of `VALUES`, and statement termination
```suggestion
            .push(" ON CONFLICT DO NOTHING;")
```

Reply:
:rocket: 

the statement termination btw. is fully optional and it's not uncommon for query builders to expect you to not include it

through it might be useful in case you forget a `reset` on a query builder. But I'm not sure if, as I haven't tried out what happens if you place multiple statements in the same query.